package.json

```diff
@@ -16,7 +16,7 @@
     "type-check:cypress": "tsc --project cypress/tsconfig.json",
     "lint": "eslint . --ext .js,.jsx,.ts,.tsx --max-warnings 0 --report-unused-disable-directives",
     "lint:fix": "pnpm lint -- --fix",
-    "prettier-base": "prettier . --ignore-unknown",
+    "prettier-base": "prettier . --ignore-unknown --cache --loglevel warn",
```

Comment:
I'm probably doing something wrong, I still get the scrolling file contents output, even with `--loglevel warn`. 🙈

Reply:
I don't see any file content output with the loglevel set to warn:

<img width="825" alt="CleanShot 2022-08-12 at 14 38 21@2x" src="https://user-images.githubusercontent.com/2677072/184355508-27ac9cdc-b68a-41a8-b09c-616e8cb32058.png">

<img width="825" alt="CleanShot 2022-08-12 at 14 38 57@2x" src="https://user-images.githubusercontent.com/2677072/184355593-6ef33efb-e33c-4b1f-85e4-557c29dd93db.png">

Reply:
Did you use the command in my suggestion? Maybe I messed up when writing the `loglevel` flag?

Reply:
I ran this locally in your branch!